xml/System.Threading.Tasks/Task.xml

```diff
@@ -5340,9 +5340,9 @@ This member is an explicit interface member implementation. It can be used only
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="0" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="cancellationToken">The <see cref="P:System.Threading.Tasks.Task.CancellationToken" /> to monitor for a cancellation request.</param>
+        <summary>Gets a <see cref="T:System.Threading.Tasks.Task" /> that will complete when this <see cref="T:System.Threading.Tasks.Task" /> completes or when the specified <see cref="P:System.Threading.Tasks.Task.CancellationToken" /> has cancellation requested.</summary>
+        <returns>The <see cref="T:System.Threading.Tasks.Task" /> representing the asynchronous wait.  It may or may not be the same instance as the current instance.</returns>
```

Comment:
nit: extra space

```suggestion
        <returns>The <see cref="T:System.Threading.Tasks.Task" /> representing the asynchronous wait. It may or may not be the same instance as the current instance.</returns>
```